packages/components/psammead-copyright/package.json

```diff
@@ -28,6 +28,7 @@
     "figure"
```

Comment:
Could you please also increase the version in the `package.json` & update the `package-lock.json`? This way the changes will be picked up by Jenkins upon merge and published to npm. So the readme will be updated here: https://www.npmjs.com/package/@bbc/psammead-copyright

We have a `CHANGELOG.md` file for this package, where you can add a brief description of your changes.

Reply:
Thanks for the reminder @sareh, completely forgot about it. Should be all good now :)